src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -113,9 +113,9 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
         type: InfoMessageType.WARNING,
         message: InfoMessage.TASK_NO_LONGER_AVAILABLE,
       });
-      if (handledStatus === 400) {
-        this.refreshTasks();
-      }
+    }
+    if (handledStatus === 400) {
```

Comment:
What does this change accomplish? `handleStatus === 400` can only be true if the condition above it is also true (`handleStatus > 0`) so this simply adds another check that is guaranteed to evaluate to false if the first one is false. Or am I missing something? 🤷

Reply:
Agreed. Done now.